src/test/scala/nl.knaw.dans.easy.deposit/authentication/SessionSpec.scala

```diff
@@ -129,13 +129,14 @@ class SessionSpec extends TestSupportFixture with ServletFixture with ScalatraSu
       uri = "/auth/login",
       params = Seq(("login", "foo"), ("password", "bar"))
     ) {
-      status shouldBe OK_200
-      body shouldBe "signed in"
-      header("Content-Type") shouldBe "text/plain;charset=UTF-8"
+      status shouldBe NO_CONTENT_204
+      body shouldBe ""
+      header("Content-Type") shouldBe "text/html;charset=UTF-8"
```

Comment:
Why is the content type changed to `text/html`?

Reply:
The content type changed when I changed the method from `put` to `post`. The whole check is actually not needed now when the body should be empty, so I deleted the whole line.